ceph/ceph.py

```diff
@@ -1615,19 +1622,34 @@ def open_firewall_port(self, port, protocol):
             self.exec_command(sudo=True, cmd="systemctl enable firewalld")
             self.exec_command(sudo=True, cmd="systemctl start firewalld")
             self.exec_command(sudo=True, cmd="systemctl status firewalld")
-            self.exec_command(
-                sudo=True,
-                cmd="firewall-cmd --zone=public --add-port={port}/{protocol}".format(
-                    port=port, protocol=protocol
-                ),
-            )
-            self.exec_command(
-                sudo=True,
-                cmd="firewall-cmd --zone=public --add-port={port}/{protocol} --permanent".format(
-                    port=port, protocol=protocol
-                ),
-            )
-        if self.pkg_type == "deb":
+
+        elif self.pkg_type == "deb":
+            # Ubuntu section stub
+            pass
+
+    def open_firewall_port(self, port, protocol):
+        """
+        Opens firewall port on the node
+
+        Args:
+            port(str|int|list): port(s) to be enabled
+            protocol(str): protocol
+        """
+        if self.pkg_type == "rpm":
+            cmd = "firewall-cmd --zone=public "
+
+            if type(port) in (str, int):
```

Comment:
IMO, it is a good practice to use `isinstance` instead of `type`. For example

```
if isinstance(port, (str, int)):
```

Here, I agree that it has no impact knowing about the argument.